src/controller/user/routes.ts

```diff
@@ -1,9 +1,10 @@
-import { FastifyInstance } from 'fastify'
-import { getUserById } from './getUserById'
-import { getUserByEmail } from './getUserByEmail'
+import { FastifyInstance } from "fastify";
+import { getUserById } from "./getUserById";
+import { getUserByEmail } from "./getUserByEmail";
+import { registerUser } from "./registerUser";
 
 export async function userRoutes(app: FastifyInstance) {
-
-  app.get('/user/:id', getUserById)
-  app.get('/user', getUserByEmail)
+  app.post("/users", registerUser);
+  app.get("/user/:id", getUserById);
+  app.get("/user", getUserByEmail);
```

Comment:
It seems like you didn't set up the linting as we discussed in the setup meetings. Could you set up ESLint the same way the rest of the team is working?